src/components/Form/index.tsx

```diff
@@ -174,7 +174,7 @@ export const Form = ({
       setValue('object', '')
     } else {
       setValue('stars', null)
-      setValue('aspect', '')
+      setValue('aspect', [])
```

Comment:
yeah so we want this, but it requires other changes actually on front end - we have to create multiple claims and sign and submit all, in front end

because back end can't sign a claim

Reply:
yeh we know we make it till we change in the data model